tests/badge.spec.ts

```diff
@@ -0,0 +1,68 @@
+import { test, expect } from '@playwright/test';
+import { Children } from 'react';
+
+test('Badge E2E Tests', async ({ page }) => {
+  await page.goto('/test.html');
+
+  // Gray Badge Test
+
+  const grayBadge = page.getByTestId("gray")
+  
+  await expect(grayBadge).toBeVisible()
+  await expect(grayBadge).toHaveText("Inactive")
+
+  const childGrayBadge = grayBadge.locator('small').nth(0)
+  
+  await expect(childGrayBadge).toHaveCSS("background-color", "rgb(229, 231, 235)")
```

Comment:
Similar comment here as the other test. I don't think we should be testing for specific CSS values.

Reply:
I see that we're trying to test the different states of this pill. I'd instead recommend testing the element's state via running assertions on attributes

Reply:
I'm also seeing that the test criteria is specifying color, so if we want to stay true to that then my alternate recommendation is that we should move these colors into the tailwind config file, add them into our theme, and use those theme colors in the pill element + import those theme colors here, so that when design updates happen we won't have to edit both the component and tests.

Reply:
I moved the colors needed for the badge variants into the tailwind config.  They render correctly when I run the storybook page locally.  However, when I run my playwright tests, the colors I added to the tailwind config weirdly don't render.  Do you happen to have any suggestions?

Reply:
Oh that's probably because the "playground" has its own tailwind config. Getting that working should be as simple as exporting that theme object from the main config into the playground config.

Reply:
Thank you!!!! That totally did the trick!